project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,167 @@
+        self._expect_ok(task, self.user_dev, self.stage_assigned_or_sales)
+        self._expect_ok(task, self.user_sales, self.stage_assigned_or_sales)
+        self._expect_fail(task, self.user_pm, self.stage_assigned_or_sales)
+
+    def test_pm_or_group(self):
+        task = self._make_task(self.user_dev)
+        self._expect_ok(task, self.user_pm, self.stage_pm_or_sales)
+        self._expect_ok(task, self.user_sales, self.stage_pm_or_sales)
+        self._expect_fail(task, self.user_dev, self.stage_pm_or_sales)
+
+    def test_superuser_bypass(self):
+        """Super‑user must bypass every restriction."""
```

Ruff (0.11.9)

Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)